README.md

```diff
@@ -1,2 +0,0 @@
-# dynamodb-sink
```

Comment:
I dont know the exact role of rst but if you dont have the .md file no description is shown on the repo home page